src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -173,7 +173,7 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
 
         const auto compressorSequence = thisChiller.Representation->performance.performance_map_cooling.grid_variables.compressor_sequence_number;
         // minmax_element is sound but perhaps overkill; as sequence numbers are required by A205 to be in ascending order
-        const auto minmaxSequenceNum = std::minmax_element(compressorSequence.begin(), compressorSequence.end());
+        const auto &minmaxSequenceNum = std::minmax_element(compressorSequence.begin(), compressorSequence.end());
```

Comment:
Code compiles and runs with no diffs but I still question whether correct.

Reply:
I think no `&` in this case since `minmax_element` returns a `std::pair` and the copy will presumably be eliminated by RVO.